src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -50,8 +50,8 @@ private TypicalPersons() {} // prevents instantiation
     /**
      * Returns an {@code Deck} with all the typical persons.
      */
-    public static Deck getTypicalAddressBook() {
-        Deck ab = new Deck();
+    public static MasterDeck getTypicalAddressBook() {
+        MasterDeck ab = new MasterDeck();
```

Comment:
Small thing, but shouldn't be ab here since now its MasterDeck instead of AddressBook